### PR TITLE
Captain's Turtleneck and skirt variant

### PR DESCRIPTION
**Pull request by thesaladmancer**
## About the PR

Added new Turtleneck jumpsuit and jumpskirt to the captain's dresser and loadout.

## Why / Balance

I thought it would be nice for Captain to get a turtleneck outfit to fit with every other command member, since it was kinda strange that they didn't have one before (HoP as well, but they recently got one too.)

## Technical details

Added sprites for the jumpsuit and jumpskirt in `capturtleneck.rsi` and `capturtleskirt.rsi`, along with their associated `meta.json` files.
Added entries for the jumpsuit and jumpskirt to `jumpsuits.yml` and `jumpskirts.yml` respectively.
Added the jumpsuit and jumpskirt to the Uniform Printer's `lathe.yml` entry
Added the jumpsuit and jumpskirt Uniform Printer recipe to `clothing.yml` in `/Resources/Prototypes/_Harmony/Recipes`
Added the jumpsuit and jumpskirt to the Captain's `captain.yml` file in `/Resources/Prototypes/_Harmony/Loadouts/Jobs/Command`
Added the jumpsuit and jumpskirt to the Captain's jumpsuit loadout group in `loadout_groups.yml`.

## Media

turtleneck
turtleskirt
image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

:cl:
- add: Added Captain's Turtleneck and skirt variant.

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml [hunk 1]
```diff
@@ -20,6 +20,17 @@
...
+  parent: [ClothingUniformBase, BaseCommandContraband]
+  id: ClothingUniformJumpskirtCaptainTurtleneck
+  name: captain's turtleneck
+  description: A regal blue turtleneck finely made with mothroach silk. Fit for a captain.
+  components:
+  - type: Sprite
+    sprite: Clothing/Uniforms/Jumpskirt/capturtleskirt.rsi
+  - type: Clothing
+    sprite: Clothing/Uniforms/Jumpskirt/capturtleskirt.rsi
+
```
Comment (FluffMe) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml:33]: This entity should be moved to `Resources/Prototypes/_Harmony/Entities/Clothing/Uniforms/jumpskirts.yml`

**Comment by thesaladmancer** [comment]
> I'd move it out of the dresser and into the uniform printer, same as our hop turtleneck and skirt.

That's a pretty good idea actually, I wasn't aware of the HoP turtlenecks being in the uniform printer as the main method of obtaining them during the round. I should still keep it as a loadout option too, yeah?

File: Resources/Textures/Clothing/Uniforms/Jumpskirt/capturtleskirt.rsi/meta.json
Comment (FluffMe) [inline Resources/Textures/Clothing/Uniforms/Jumpskirt/capturtleskirt.rsi/meta.json:1]: This RSI and all its textures in it should be relocated to `Resources/Textures/_Harmony/Clothing/Uniforms/Jumpskirt/capturtleskirt.rsi`

**Comment by FluffMe** [comment]
> > I'd move it out of the dresser and into the uniform printer, same as our hop turtleneck and skirt.
> 
> That's a pretty good idea actually, I wasn't aware of the HoP turtlenecks being in the uniform printer as the main method of obtaining them during the round. I should still keep it as a loadout option too, yeah?

Correct! You can refer to #104 and #108  for implementation

**Review by FluffMe — CHANGES_REQUESTED** [review]
Entities and textures need relocating to other folders.

Once relocation is complete, and build error is fixed via uniform printer, I'll do another more thorough pass. This one is just files location without inspecting the entities.

**Review by FluffMe — APPROVED** [review]
Code looks good.

Thank you very much!

Additional screenshots for other reviewers:
**Lathe**
image
**Loadout**
image
**Item description**
image

**Review by DieselMohawk — APPROVED** [review]
Looks good

**Pull request merged**